Taskfile.yml

```
@@ -207,6 +208,28 @@ tasks:
     cmds:
       - go -C . test ./... {{.EXTRA_ARGS}}
 
+  sdk:deps:common:
+    desc: Common dependencies for SDKs
+    cmds:
+      - task: deps:cosign
+      - task: cli:compile
+
+  sdk:deps:iodc-token-generation:
+    desc: Get Fulcio OIDC token for CI
```

nit: more general deps for SDK CICD job

```suggestion
  sdk:deps:cicd:
    desc: Get SDK dependencies for CICD
```